### ISSUE TITLE
Table sort state is missing or incorrect

### ISSUE THREAD

**Issue by schne324** [dequelabs/cauldron#578]
<dl><dt>Description</dt><dd>The sorted state (ascending/descending) of table columns is missing or incorrect.
Below are the issues on Sortable table - 
1.Table column's ''Email', 'Last Name'  sort state is not being announced by screen reader.
2. Sort - Ascending /descending order information is missing. 

</dd><dt>Impact</dt><dd>Serious</dd><dt>Recommendation</dt><dd>RULE : 
The name, role, value, states, and properties of user interface components MUST be programmatically determinable by assistive technologies.

HOW TO FIX: 
Fix this issue by using ONE of the following techniques:
1. Use the aria-sort attribute values of "ascending" or "descending" or "none" on the column header to indicate the sort state of the data in a sortable column. This technique also requires that role="columnheader" be added to the <th> element.

<th role="columnheader" aria-sort="descending">
<button class="sortableColumnLabel">Name</button>
</th>

2. Add CSS clipped (visually hidden) text to the sorting element to indicate the sorted state.

<th>
<div tabindex="0" role="button" id="date-sort">
Date <span class="sr-text">sorted ascending</span>
</div>
</th>

REFERENCE: 
WAI-ARIA Recommendation: https://www.w3.org/TR/wai-aria-1.1/#aria-sort

BACKGROUND: 
States and properties are attributes used to convey essential information about an element to screen readers and other assistive technologies. Some roles require certain state and property information - such as the checked/unchecked state of a checkbox. Native HTML elements provide those required states and properties, so nothing more needs to be done. If you create a custom version of a native HTML element or a custom control or widget that does not have a native HTML equivalent, you must add the relevant states and properties using ARIA.</dd><dt>Source code</dt><dd><pre><code><th aria-sort="none" class="TableHeader" scope="col"><button class="TableHeader__sort-button" type="button">Last Name<span aria-hidden="true"><div aria-hidden="true" class="Icon Icon--sort-triangle"><svg overflow="visible" preserveAspectRatio="none" viewBox="0 0 24 24" height="24" width="24"><path d="M17.13 13.54a.634.634 0 0 0-.47-.2H7.34c-.36 0-.66.29-.67.65 0 .18.07.35.2.48l4.66 4.66c.25.26.66.27.92.02l.02-.02 4.66-4.66c.26-.25.28-.65.03-.91-.01-.02-.02-.02-.03-.02zm0-4-4.66-4.66a.649.649 0 0 0-.92-.02l-.02.02-4.66 4.66c-.25.27-.24.69.03.95.12.11.28.18.44.18h9.33c.36 0 .66-.3.66-.66a.72.72 0 0 0-.2-.47z" fill="currentColor" vector-effect="non-scaling-stroke"></path></svg></div></span></button></th></code></pre></dd><dt>Checkpoint</dt><dd>Name, Role, Value (4.1.2.a)</dd><dt>URL</dt><dd>https://cauldron.dequelabs.com/components/Table</dd></dl>

**Comment by schne324** [comment]
this issue has been fixed
